### ISSUE TITLE
No scenes opened when opening a project without compiled scenes

### ISSUE THREAD

**Issue by verdie-g** [Facepunch/sbox-issues#5346]
### Describe the bug

Opening a project when there is no compiled scene (*.scene_c), creates that file but doesn't open the scene.
![image](https://github.com/Facepunch/sbox-issues/assets/9092290/17f1ae46-7a0f-4353-b947-7fc8b0beb43d)

This is a little annoyance when sharing a project because if the the scene_c is not included it looks like there are no scenes. The recipient should know that they need to manually open the scene.

### To Reproduce

- Open [sbox-issue-5346.zip](https://github.com/Facepunch/sbox-issues/files/14898050/sbox-issue-5346.zip)
- Observe an empty scene

### Expected behavior

I would expect the scene to be opened after being compiled.

**Comment by handsomematt** [comment]
Fixed now I think, but please have a test

**Comment by verdie-g** [comment]
Hmm I just downloaded and open [sbox-issue-5346.zip](https://github.com/Facepunch/sbox-issues/files/14898050/sbox-issue-5346.zip) but it doesn't seem to open the minimal.scene.

**Comment by verdie-g** [comment]
Yeah I confirm that I created a new project with a new scene saved in the Assets folder. After deleting the `_c` and re-opening the project, the scene is compiled but not opened.
